README.md

```diff
@@ -582,6 +582,7 @@ Libraries to help manage database schemas and migrations.
 ## Filtering and Validation
 *Libraries for filtering and validating data.*
 
+* [HTMLPurifier](http://htmlpurifier.org/) - A HTML filtering library.
```

Comment:
Maybe we should point it to the [GitHub](https://github.com/ezyang/htmlpurifier) repo?